services/core/MasterDriverAgent/master_driver/agent.py

```diff
@@ -431,10 +443,10 @@ def revert_device(self, path, **kwargs):
 
     # Turn on override condition on all the devices matching the pattern.
     @RPC.export
-    def set_override_on(self, pattern, duration=1.0, failsafe_revert=True, staggered_revert=False):
-        self._set_override_on(pattern, duration, failsafe_revert, staggered_revert)
+    def set_override_on(self, pattern, duration=1.0, indefinite_duration=False, failsafe_revert=True, staggered_revert=False):
```

Comment:
Instead of adding a new argument could we just use duration <= 0 to signify an infinite duration and make 0 the default?